src/extension.ts

```diff
@@ -1,17 +1,19 @@
-'use strict';
+'use strict'
```

Comment:
switching to the dark side, huh?

Reply:
haha didn't mean to do that :)

just set up prettier for formatting